eventdata/runners/mount_searchable_snapshot_runner.py

```diff
@@ -15,10 +15,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import re
+
 class MountSearchableSnapshotRunner:
     async def __call__(self, es, params):
         repository_name = params["repository"]
         snapshot_name = params["snapshot"]
+        indices = params.get("indices", "*")
```

Comment:
the name `indices` as a param is a bit confusing to be: I'd think it allows a list containing static index names or even patterns.

Maybe we replace to `index_pattern` instead? Alternatively we can have it a list and have a default value of `["*"]` and later in line 36/37 replace the single if with something like the example below:

```
In [37]: index_patterns=["elasticlogs-2021-01-01","elasticlogs-2020-*"]

In [38]: index="elasticlogs-2020-03-03"

In [39]: for pattern in index_patterns:
    ...:     if fnmatch.fnmatch(index,pattern):
    ...:         print("issue request")
    ...: 
    ...: 
issue request
```

Reply:
I went with the index_pattern suggestion, see 5e8e87a